src/component/Router.js

```diff
@@ -1,23 +1,24 @@
 // This page uses the React Router library to make links to navigate from page to page
 import React from 'react';
-import './App.css';
-import Home from './component/Home';
-import FetchItems from './component/FetchItems';
-import AddItem from './component/AddItem';
+import '../App.css';
+// import Home from './Home';
+import FetchItems from './FetchItems';
+import AddItem from './AddItem';
 import { BrowserRouter as Router, Route, Link, Switch } from 'react-router-dom'
 
 function RouterComponent() {
+    // console.log('is this the router page?')
     return (
-        <Router>
+        <Router className="Router">
             {/* Our links. So far, we have three, though 
             that might change as the project grows */}
-            <Link to="/">Home</Link>
-            <Link to="/list">List</Link>
-            <Link to="/add">Add</Link>
+            {/* <Link to="/">Home</Link> */}
```

Comment:
Delete if no longer needed, but I did see an argument in favor of keeping the Home view?